### PR TITLE
Hotfix_tests into develop

### PR DESCRIPTION
**Pull request by Setnea**
Adding test6 version working with map_plate changes.

### CODE REVIEW

File: tests/release/configuration_settings/2017-02-14-t2.xml [hunk 1]
```diff
@@ -0,0 +1,562 @@
...
+                                <parameter key="output_path.3" value="//"/> 
+                            </parameters_by_value>
+                            <parameters_by_name>
+                                <parameter key="input_path.1" value="quantify_data_xxx_path"/>
+                                <parameter key="output_path.1" value="quantify_data_xxx_path"/> 
+                                <parameter key="input_path.2" value="wellname_id"/>
+                                <parameter key="output_path.2" value="wellname_id"/> 
+                            </parameters_by_name>
+                        </TASK>-->
+                        <TASK class="TASK_REMOVE">
```
Comment (stork119) [inline tests/release/configuration_settings/2017-02-14-t2.xml]: It should not be commented

**Review by stork119 — CHANGES_REQUESTED** [review]
Tests have commented parts.

**Comment by Setnea** [comment]
Right, I probably was testing sth; temporary commented to speed up process and forgot to uncomment before push. Gonna be more carefull next time.

**Review by stork119 — CHANGES_REQUESTED** [review]
position.name -> id

**Comment by Setnea** [comment]
Everything works fine now. May I merge?

**Review by stork119 — APPROVED** [review]

**Pull request merged**